code/parachain/frame/composable-traits/src/centauri.rs

```diff
@@ -13,7 +13,7 @@ impl Map {
 				receiver: value.receiver,
 				port: value.port.map(|x| x.to_string()),
 				channel: value.channel.map(|x| x.to_string()),
-				timeout: value.timeout,
+				timeout: value.timeout.and_then(|x| x.parse().ok()),
```

Comment:
don't we want to fail the whole memo if this fails?

Reply:
we know 100% that the value is number
```
ForwardingMemo::new_ibc_memo(
      memo_receiver,
      PortId::transfer(),
      ChannelId::new(i.channel_id),
      i.timeout.ok_or(DispatchError::Other("Timeout is none"))?.to_string(),
      i.retries.ok_or(DispatchError::Other("Retries is none"))?,
)
```

Reply:
But what if the realisation changes and someone accidentally put not a number there?

Reply:
pushed the changes. to replace string with u64. thanks.